components/core/animation/collapse.ts

```diff
@@ -4,7 +4,7 @@ import { AnimationCurves } from './animation';
 export const collapseMotion: AnimationTriggerMetadata = trigger('collapseMotion', [
   state('expanded', style({ height: '*' })),
   state('collapsed', style({ height: 0, overflow: 'hidden' })),
-  state('hidden', style({ height: 0, display: 'none' })),
+  state('hidden', style({ height: 0 })),
```

Comment:
This would result in two borders between two panels.

![X4gwA5grIr](https://user-images.githubusercontent.com/12122021/55464984-cd0a4f00-562e-11e9-8276-9b0d8932ba05.png)